cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -97,6 +97,7 @@
         <transition on="success" to="sendTicketGrantingTicket" />
         <transition on="authenticationFailure" to="handleAuthenticationFailure" />
         <transition on="error" to="beforeLogin" />
+        <transition on="invalidVerificationKey" to="casLoginInvalidVerificationKeyView" />
```

Comment:
The event "invalidVerificationKey" just redirects user to the end state "casLoginInvalidVerificationKeyView".

![casinvalidverificationkeyview](https://user-images.githubusercontent.com/3750414/44921936-f1f1b900-ad11-11e8-85d5-887301a677f4.png)